apps/explorer/src/ui/Tabs.tsx

```diff
@@ -55,13 +55,16 @@ const TabsList = forwardRef<
 		fullWidth?: boolean;
 		disableBottomBorder?: boolean;
 		lessSpacing?: boolean;
+		dynamicSpacing?: boolean;
 	}
->(({ fullWidth, disableBottomBorder, lessSpacing, ...props }, ref) => (
+>(({ fullWidth, disableBottomBorder, lessSpacing, dynamicSpacing, ...props }, ref) => (
 	<TabsPrimitive.List
 		ref={ref}
 		className={clsx(
 			'flex items-center border-gray-45',
-			lessSpacing ? 'gap-3' : 'gap-6',
+			lessSpacing && !dynamicSpacing && 'gap-3',
```

Comment:
this should be moved into `variants` either as part of the base styles or with a new `spacing` variant that allows you to control the gap between tabs. I also think the responsive styles should be baked into the core styles as I don't see a case where we wouldn't want the gap to change with the screen size

Reply:
@mamos-mysten updated